### PR TITLE
logout functionaliteit

### PR DESCRIPTION
**Pull request by pidip47**

### CODE REVIEW

File: client/src/components/global/Topbar.tsx [hunk 1]
```diff
@@ -113,10 +114,15 @@ export default function Topbar(): JSX.Element {
...
-										<Button onClick={(): Promise<void> => validate(user)}
+										// <Button onClick={(): Promise<void> => validate(user)}
+										// 	color='secondary' startIcon={<PersonIcon />}
+										// 	variant='contained'>
+										// 	My Account
+										// </Button>
+										<Button onClick={(): Promise<void> => signout(user)}
 											color='secondary' startIcon={<PersonIcon />}
 											variant='contained'>
-											My Account
```
Comment (ManoloTonto1) [inline client/src/components/global/Topbar.tsx:119]: So now we have a logout button, How will the user check their account? I think its a better idea to leave the account button and let it drop down to a few options. Check the figma designs I made a few for each possible gebruiker.

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]

**Comment by ManoloTonto1** [comment]
So the DONATE token got changed this week by the donation server and was incorporated in an other pr. maybe consider merging main to this so you don't get the error anymore.

File: client/src/components/global/Topbar.tsx (hunk 1 above)
Comment (pidip47) [reply client/src/components/global/Topbar.tsx:119]: it was for testing if it worked, forgot to remove it whoops

**Comment by ManoloTonto1** [comment]
@pidip47 any changes on this?

**Review by ManoloTonto1 — APPROVED** [review]
Manolotonto Approves 🚀 !!

**Pull request merged**